modules/data_team_helper/README.md

```diff
@@ -0,0 +1,29 @@
+# Data Team Helper
+
+## Purpose
+
+The Data Team Helper module is intended to provide a dashboard to
+make it easier for the data entry staff of a LORIS project to access
+all the unfinished data entry.
+
+## Scope
+
+The data team helper provides links to unfinished data in other
+LORIS modules in a centralized place. In particular, it links to
+open feedback, unresolved conflict, and candidate instruments where
+the data entry (or double data entry) has not yet been completed.
+
```

Comment:
It also provides  an overview graph indicating the data-entry progress/status across All Candidates/ Visits/Instruments/Sites/Projects